recipes/openssl/1.x.x/conanfile.py

```diff
@@ -699,6 +714,7 @@ def build(self):
                 else:
                     self._patch_configure()
                     self._patch_makefile_org()
+            with self._make_context():
```

Comment:
I think this needs to be indented an extra level.

Reply:
Thanks for noting this!